cmd/helm/status.go

```diff
@@ -116,6 +116,9 @@ func (s statusPrinter) WriteTable(out io.Writer) error {
 	fmt.Fprintf(out, "NAMESPACE: %s\n", s.release.Namespace)
 	fmt.Fprintf(out, "STATUS: %s\n", s.release.Info.Status.String())
 	fmt.Fprintf(out, "REVISION: %d\n", s.release.Version)
+	if len(s.release.Info.Description) > 0 {
```

Comment:
we usually use following to test for empty string: `if s.release.Info.Description != "" {`